### DISCUSSION TITLE
Autoplay via url

### DISCUSSION THREAD

**Discussion by Kivylius** [jellyfin/jellyfin#8380] [General]
Can we add a way to autoplay the video via a url param? Like we can share URL that link to that exect movie, and we can add a autoplay flag, just like youtube had.

`http://192.168.0.2:8096/web/index.html#!/details?id=123&context=home&serverId=122&autoplay=true`.

This allows us to share url with friends and the movie to start automaticly. 
This is also great for external tools that need to deep link to a movie to play, instead of the movie page.

I would find that useful, but not sure if there is issues with that.

**Comment by AnonymousWebHacker** [comment]
https://streamlink.github.io
